locales/hr.json

```diff
@@ -3,21 +3,17 @@
         "edit": "Uređivanje",
         "cancel": "Prekid",
         "continue_shopping": "Nastavak kupovine",
-        "back_to_checkout": "Povratak na blagajnu",
-        "checkout": "Blagajna",
+        "back_to_checkout": "Povratak na završetak narudžbe",
+        "checkout": "Završetak narudžbe",
         "apply": "Primjena",
         "type_address": "Unos adrese",
         "use_this_address": "Koristiti ovu adresu",
         "back_to_store": "Povratak u trgovinu",
         "close_cart": "Zatvoriti košaricu",
         "show": "Prikazivanje",
         "hide": "Skrivanje",
-        "apply_changes": "Primjena svih izmjena",
-        "yes_use_it": "Da, koristiti",
-        "save_changes": "Spremi izmjene",
```

Comment:
Please make sure you add back those phrases:

- save_changes
- back_to_orders
- change_password
- add